packages/deployment/upgrade-test/Dockerfile

```diff
@@ -95,8 +95,8 @@ ARG DEST_IMAGE
 FROM ghcr.io/agoric/agoric-sdk:35 as propose-agoric-upgrade-11
 # This default UPGRADE_INFO_11 is to test core proposals like the network vat.
 # TODO: Maybe replace with a Zoe core proposal, or remove when other paths test it.
-ARG BOOTSTRAP_MODE UPGRADE_INFO_11='{"coreProposals":["@agoric/builders/scripts/vats/init-network.js"]}'
-ENV THIS_NAME=propose-agoric-upgrade-11 UPGRADE_TO=agoric-upgrade-11 UPGRADE_INFO=${UPGRADE_INFO_11} BOOTSTRAP_MODE=${BOOTSTRAP_MODE}
+ARG BOOTSTRAP_MODE
+ENV THIS_NAME=propose-agoric-upgrade-11 UPGRADE_TO=agoric-upgrade-11
```

Comment:
It should still take the arg, just not have a default value.

The default value is being removed in https://github.com/Agoric/agoric-sdk/pull/8287/files#diff-b939489d9628276f8e9dfb17c4d415a4e83668fe8d05af26858ce1870446bea9 so let's just stick with that one.